test_data_choosers failing sporadically on CI

Sometimes, we get the following test failure on CI. I haven't been able to reproduce locally:

```

Run GabrielBB/xvfb-action@v1
C:\hostedtoolcache\windows\Python\3.11.2\x64\python.exe -m pytest -s -p no:faulthandler --color=yes
============================= test session starts =============================
platform win32 -- Python 3.11.2, pytest-7.2.2, pluggy-1.0.0
PyQt5 5.15.9 -- Qt runtime 5.15.2 -- Qt compiled 5.15.2
rootdir: D:\a\napari-imagej\napari-imagej, configfile: pyproject.toml
plugins: napari-0.4.17, napari-plugin-engine-0.2.0, npe2-0.6.2, cov-4.0.0, env-0.8.1, qt-4.2.0
collected 148 items

tests\test_scripting.py .
tests\test_settings.py .
tests\types\test_converters.py ...................................
tests\types\test_enums.py .
tests\types\test_type_conversions.py ......
tests\types\test_widget_mappings.py ................
tests\utilities\test_module_utils.py ................................
tests\widgets\test_info_bar.py .
tests\widgets\test_menu.py ..sssF...s...
tests\widgets\test_napari_imagej.py ............
tests\widgets\test_parameter_widgets.py ...................
tests\widgets\test_result_runner.py ..
tests\widgets\test_result_tree.py .......
tests\widgets\test_searchbar.py ..

================================== FAILURES ===================================
_____________________________ test_data_choosers ______________________________

asserter = <function asserter.<locals>.assertFunc at 0x00000146C1A651C0>
qtbot = <pytestqt.qtbot.QtBot object at 0x00000146C1115D50>
ij = <java object 'net.imagej.ImageJ'>
gui_widget_chooser = <napari_imagej.widgets.menu.NapariImageJMenu object at 0x00000146A0A52320>

    @pytest.mark.skipif(TESTING_HEADLESS, reason="Only applies when not running headlessly")
    def test_data_choosers(asserter, qtbot, ij, gui_widget_chooser):
        button_to: ToIJButton = gui_widget_chooser.to_ij
        button_from: FromIJButton = gui_widget_chooser.from_ij
        assert not button_to.isEnabled()
        assert not button_from.isEnabled()
    
        # Show the button
        qtbot.mouseClick(gui_widget_chooser.gui_button, Qt.LeftButton, delay=1)
>       asserter(button_to.isEnabled)

tests\widgets\test_menu.py:273: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

func = <built-in method isEnabled of ToIJButton object at 0x00000146A0A52200>

    def assertFunc(func: Callable[[], bool]):
        # Let things run for up to a minute
>       qtbot.waitUntil(func, timeout=timeout)
E       pytestqt.exceptions.TimeoutError: waitUntil timed out in 60000 milliseconds

tests\conftest.py:29: TimeoutError
============================== warnings summary ===============================
C:\hostedtoolcache\windows\Python\3.11.2\x64\Lib\site-packages\vispy\app\backends\_qt.py:93
  C:\hostedtoolcache\windows\Python\3.11.2\x64\Lib\site-packages\vispy\app\backends\_qt.py:93: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    if LooseVersion(QT_VERSION_STR) >= '5.4.0':

C:\hostedtoolcache\windows\Python\3.11.2\x64\Lib\site-packages\setuptools\_distutils\version.py:346
  C:\hostedtoolcache\windows\Python\3.11.2\x64\Lib\site-packages\setuptools\_distutils\version.py:346: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    other = LooseVersion(other)

tests/types/test_converters.py::test_ellipse_layer_to_mask
tests/types/test_converters.py::test_rectangle_layer_to_mask_box
tests/types/test_converters.py::test_line_layer_to_mask
tests/types/test_converters.py::test_line_layer_to_mask
tests/types/test_converters.py::test_path_layer_to_mask
tests/types/test_converters.py::test_path_layer_to_mask
tests/types/test_converters.py::test_path_layer_to_mask
tests/types/test_converters.py::test_multiple_layer_to_masks
tests/types/test_converters.py::test_multiple_layer_to_masks
  C:\hostedtoolcache\windows\Python\3.11.2\x64\Lib\site-packages\scyjava\_convert.py:711: DeprecationWarning: `np.bool8` is a deprecated alias for `np.bool_`.  (Deprecated NumPy 1.24)
    JBoolean: np.bool8,

tests/widgets/test_napari_imagej.py::test_event_subscriber_registered
  C:\hostedtoolcache\windows\Python\3.11.2\x64\Lib\site-packages\_pytest\python.py:199: PytestReturnNotNoneWarning: Expected None, but tests/widgets/test_napari_imagej.py::test_event_subscriber_registered returned True, which will be an error in a future version of pytest.  Did you mean to use `assert` instead of `return`?
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ===========================
FAILED tests/widgets/test_menu.py::test_data_choosers - pytestqt.exceptions.TimeoutError: waitUntil timed out in 60000 milliseconds
====== 1 failed, 143 passed, 4 skipped, 12 warnings in 183.93s (0:03:03) ======
```

We might want to consider disabling the test on CI, if we can't get to the bottom of this.

--- a/src/napari_imagej/widgets/menu.py
+++ b/src/napari_imagej/widgets/menu.py
@@ -17,7 +17,7 @@ from qtpy.QtWidgets import QHBoxLayout, QMessageBox, QPushButton, QWidget
 from scyjava import is_arraylike
 
 from napari_imagej import settings
-from napari_imagej.java import ij, java_signals, jc, log_debug
+from napari_imagej.java import ij, java_signals, jc
 from napari_imagej.resources import resource_path
 
 
@@ -43,8 +43,6 @@ class NapariImageJMenu(QWidget):
             self.gui_button.clicked.connect(self.gui_button.disable_popup)
         else:
             self.gui_button.clicked.connect(self._showUI)
-            self.gui_button.clicked.connect(lambda: self.to_ij.setEnabled(True))
-            self.gui_button.clicked.connect(lambda: self.from_ij.setEnabled(True))
 
     @property
     def gui(self) -> "jc.UserInterface":
@@ -142,7 +140,6 @@ class ToIJButton(QPushButton):
         super().__init__()
         self.viewer = viewer
 
-        self.setEnabled(False)
         icon = QColoredSVGIcon.from_resources("long_right_arrow")
         self.setIcon(icon.colored(theme=viewer.theme))
         if settings["choose_active_layer"].get():
@@ -161,7 +158,7 @@ class ToIJButton(QPushButton):
         if active_layer:
             self._show(active_layer)
         else:
-            log_debug("There is no active layer to export to ImageJ2")
+            self.handle_no_choices()
 
     def send_chosen_layer(self):
         # Get Layer choice
@@ -170,7 +167,9 @@ class ToIJButton(QPushButton):
             layer={"annotation": Layer, "options": {"choices": get_layers}},
         )
         # Parse choices for the layer
-        if choices is not None:
+        if choices is None:
+            self.handle_no_choices()
+        else:
             layer = choices["layer"]
             if isinstance(layer, Layer):
                 # Pass the relevant data to ImageJ2
@@ -181,13 +180,18 @@ class ToIJButton(QPushButton):
         # TODO: Use EventQueue.invokeLater scyjava wrapper, once it exists
         ij().thread().queue(lambda: ij().ui().show(ij().py.to_java(layer)))
 
+    def handle_no_choices(self):
+        RichTextPopup(
+            rich_message="There is no active window to export to ImageJ!",
+            exec=True,
+        )
+
 
 class FromIJButton(QPushButton):
     def __init__(self, viewer: Viewer):
         super().__init__()
         self.viewer = viewer
 
-        self.setEnabled(False)
         icon = QColoredSVGIcon.from_resources("long_left_arrow")
         self.setIcon(icon.colored(theme=viewer.theme))
         if settings["choose_active_layer"].get():
@@ -215,7 +219,9 @@ class FromIJButton(QPushButton):
             title="Send layers to napari",
             data={"annotation": Enum, "options": {"choices": names}},
         )
-        if choices is not None:
+        if choices is None:
+            self.handle_no_choices
+        else:
             # grab the chosen name
             name = choices["data"]
             display = ij().display().getDisplay(name)
@@ -240,10 +246,10 @@ class FromIJButton(QPushButton):
         # TODO: simplify to no-args once
         # https://github.com/imagej/imagej-legacy/pull/287 is merged.
         view = ids.getActiveDatasetView(ids.getActiveImageDisplay())
-        if view is None:
-            log_debug("There is no active window to export to napari")
-            return
-        self._add_image(view)
+        if view is not None:
+            self._add_image(view)
+        else:
+            self.handle_no_choices()
 
     def _add_image(self, view: Union["jc.Dataset", "jc.DatasetView"]):
         # Get the stuff needed for a new layer
@@ -256,6 +262,12 @@ class FromIJButton(QPushButton):
             self.viewer.add_image(data=py_image, name=name)
         else:
             raise ValueError(f"{view} cannot be displayed in napari!")
+
+    def handle_no_choices(self):
+        RichTextPopup(
+            rich_message="There is no active window to export to napari!",
+            exec=True,
+        )
 
 
 class GUIButton(QPushButton):

--- a/tests/widgets/test_menu.py
+++ b/tests/widgets/test_menu.py
@@ -144,13 +144,8 @@ def test_widget_layout(gui_widget: NapariImageJMenu):
     assert isinstance(subwidgets[0], QHBoxLayout)
 
     assert isinstance(subwidgets[1], FromIJButton)
-    assert not subwidgets[1].isEnabled()
-
     assert isinstance(subwidgets[2], ToIJButton)
-    assert not subwidgets[2].isEnabled()
-
     assert isinstance(subwidgets[3], GUIButton)
-
     assert isinstance(subwidgets[4], SettingsButton)
 
 
@@ -208,7 +203,7 @@ def test_GUIButton_layout_headless(popup_handler, gui_widget: NapariImageJMenu):
 )
 def test_active_data_send(asserter, qtbot, ij, gui_widget: NapariImageJMenu):
     button: ToIJButton = gui_widget.to_ij
-    assert not button.isEnabled()
+    assert button.isEnabled()
 
     # Show the button
     qtbot.mouseClick(gui_widget.gui_button, Qt.LeftButton, delay=1)
@@ -239,7 +234,7 @@ def test_active_data_send(asserter, qtbot, ij, gui_widget: NapariImageJMenu):
 )
 def test_active_data_receive(asserter, qtbot, ij, gui_widget: NapariImageJMenu):
     button: FromIJButton = gui_widget.from_ij
-    assert not button.isEnabled()
+    assert button.isEnabled()
 
     # Show the button
     gui_widget.gui_button.clicked.emit()
@@ -265,13 +260,8 @@ def test_active_data_receive(asserter, qtbot, ij, gui_widget: NapariImageJMenu):
 def test_data_choosers(asserter, qtbot, ij, gui_widget_chooser):
     button_to: ToIJButton = gui_widget_chooser.to_ij
     button_from: FromIJButton = gui_widget_chooser.from_ij
-    assert not button_to.isEnabled()
-    assert not button_from.isEnabled()
-
-    # Show the button
-    qtbot.mouseClick(gui_widget_chooser.gui_button, Qt.LeftButton, delay=1)
-    asserter(button_to.isEnabled)
-    asserter(button_from.isEnabled)
+    assert button_to.isEnabled()
+    assert button_from.isEnabled()
 
     # Add a layer
     sample_data = numpy.ones((100, 100, 3))

